src/components/Chat/ChatMessage.tsx

```diff
@@ -2,19 +2,34 @@ import * as React from 'react'
 import * as PropTypes from 'prop-types'
 import * as cx from 'classnames'
 
-import { childrenExist, createShorthandFactory, customPropTypes, UIComponent } from '../../lib'
-import { ComponentVariablesInput, IComponentPartStylesInput } from '../../../types/theme'
+import {
+  childrenExist,
+  createShorthandFactory,
+  customPropTypes,
+  UIComponent,
+  IRenderResultConfig,
+  pxToRem,
+} from '../../lib'
+import {
+  ComponentVariablesInput,
+  IComponentPartStylesInput,
+  ComponentPartStyle,
+} from '../../../types/theme'
 import { Extendable, ReactChildren, ItemShorthand } from '../../../types/utils'
 import Avatar from '../Avatar'
+import Layout from '../Layout'
+import Text from '../Text'
 
 export interface IChatMessageProps {
   as?: any
+  author?: string
   avatar?: ItemShorthand
   children?: ReactChildren
   className?: string
   content?: any
   mine?: boolean
   styles?: IComponentPartStylesInput
+  timestamp?: string
   variables?: ComponentVariablesInput
```

Comment:
Why don't we make the text and the author shorthands for the Text component?

Reply:
yeah - have discussed with @mnajdova, here proposal is essentially to introduce `author` and `timestamp` shorthands (slots) for the `ChatMessage` component, and use `Text` as a default component for these.

Totally support this idea 👍